src/Controller/Host/CreateEventController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Controller\Host;
+
+use App\Entity\Experience;
+use App\Repository\EventRepository;
+use App\Request\EventRequest;
+use App\Service\EventService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CreateEventController extends AbstractController
+{
+    #[Route('/host/{id}/create/event', name: 'app_host_create_event', methods: ['POST'])]
+    public function createEvent(Experience $experience, EventRepository $repository, EventRequest $request, EventService $eventService)
```

Comment:
Please move service dependencies(i.e. `Experience` and `EventRepository`) to service.